site/app/libraries/database/DatabaseQueries.php

```diff
@@ -3204,6 +3223,21 @@ public function getPeerGradingAssignment($gradeable_id) {
         }
         return $return;
     }
+    
+    /**
+     * Adds an assignment for someone to get all the peer feedback for a given gradeable
+     *
+     * @param string $gradeable_id
+     */
+    public function getPeerFeedback($gradeable_id) {
```

Comment:
Let's rename this `getAllPeerFeedback` or equivalent. I initially thought it would get the peer feedback for a specific grader based on the name.